index.html

```diff
@@ -989,49 +980,43 @@
           <ol>
             <li>Let <em>P</em> be a new <a>Promise</a>.
             </li>
-            <li>Return <em>P</em>.
+            <li>Return <em>P</em>, but continue running these steps in
+            parallel.
```

Comment:
You can ignore this comment, as long as the meaning of "in parallel" is clarified.